<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="49" passed="1" failed="4" skipped="44">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-02-12T17:08:06Z" name="Parallel Suite" finished-at="2020-02-12T17:21:37Z" duration-ms="811004">
    <groups>
    </groups>
    <test started-at="2020-02-12T17:08:06Z" name="test4" finished-at="2020-02-12T17:13:20Z" duration-ms="314190">
      <class name="com.thed.zapi.cloud.sample.SeleniumAmazonTest3">
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@2133814f]" started-at="2020-02-12T17:08:06Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:11:47Z" duration-ms="220576" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@stsexports.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-stsexports]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.seleniumUI(SeleniumAmazonTest3.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.SeleniumAmazonUserTest(SeleniumAmazonTest3.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@2133814f]" started-at="2020-02-12T17:11:47Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:12:24Z" duration-ms="37213" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@stsexports.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-stsexports]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.seleniumUI(SeleniumAmazonTest3.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.SeleniumAmazonUserTest(SeleniumAmazonTest3.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@2133814f]" started-at="2020-02-12T17:12:24Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:13:20Z" duration-ms="56351" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@stsexports.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-stsexports]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
      </class> <!-- com.thed.zapi.cloud.sample.SeleniumAmazonTest3 -->
    </test> <!-- test4 -->
    <test started-at="2020-02-12T17:08:06Z" name="test5" finished-at="2020-02-12T17:16:45Z" duration-ms="518934">
      <class name="com.thed.zapi.cloud.sample.SeleniumAmazonTest5">
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest5@6b19b79]" started-at="2020-02-12T17:08:06Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:09:28Z" duration-ms="82281" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@trusoft.xyz]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-trusoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.seleniumUI(SeleniumAmazonTest5.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.SeleniumAmazonUserTest(SeleniumAmazonTest5.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest5@6b19b79]" started-at="2020-02-12T17:09:28Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:10:38Z" duration-ms="69095" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@trusoft.xyz]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-trusoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.seleniumUI(SeleniumAmazonTest5.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.SeleniumAmazonUserTest(SeleniumAmazonTest5.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest5@6b19b79]" started-at="2020-02-12T17:10:38Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:11:15Z" duration-ms="37122" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@trusoft.xyz]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-trusoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.seleniumUI(SeleniumAmazonTest5.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.SeleniumAmazonUserTest(SeleniumAmazonTest5.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest5@6b19b79]" started-at="2020-02-12T17:11:15Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:12:02Z" duration-ms="47374" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@trusoft.xyz]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-trusoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.seleniumUI(SeleniumAmazonTest5.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.SeleniumAmazonUserTest(SeleniumAmazonTest5.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest5@6b19b79]" started-at="2020-02-12T17:12:02Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:12:39Z" duration-ms="37209" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@trusoft.xyz]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-trusoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.seleniumUI(SeleniumAmazonTest5.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.SeleniumAmazonUserTest(SeleniumAmazonTest5.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest5@6b19b79]" started-at="2020-02-12T17:12:39Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:13:25Z" duration-ms="45231" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@trusoft.xyz]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-trusoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.seleniumUI(SeleniumAmazonTest5.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.SeleniumAmazonUserTest(SeleniumAmazonTest5.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest5@6b19b79]" started-at="2020-02-12T17:13:25Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:13:55Z" duration-ms="30738" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@trusoft.xyz]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-trusoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.seleniumUI(SeleniumAmazonTest5.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.SeleniumAmazonUserTest(SeleniumAmazonTest5.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest5@6b19b79]" started-at="2020-02-12T17:13:55Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:14:24Z" duration-ms="28909" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@trusoft.xyz]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-trusoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.seleniumUI(SeleniumAmazonTest5.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.SeleniumAmazonUserTest(SeleniumAmazonTest5.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest5@6b19b79]" started-at="2020-02-12T17:14:24Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:14:54Z" duration-ms="29833" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@trusoft.xyz]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-trusoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.seleniumUI(SeleniumAmazonTest5.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.SeleniumAmazonUserTest(SeleniumAmazonTest5.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest5@6b19b79]" started-at="2020-02-12T17:14:54Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:15:24Z" duration-ms="30362" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@trusoft.xyz]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-trusoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.seleniumUI(SeleniumAmazonTest5.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.SeleniumAmazonUserTest(SeleniumAmazonTest5.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest5@6b19b79]" started-at="2020-02-12T17:15:24Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:16:18Z" duration-ms="53199" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@trusoft.xyz]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-trusoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.seleniumUI(SeleniumAmazonTest5.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.SeleniumAmazonUserTest(SeleniumAmazonTest5.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest5@6b19b79]" started-at="2020-02-12T17:16:18Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:16:45Z" duration-ms="27504" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@trusoft.xyz]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-trusoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.seleniumUI(SeleniumAmazonTest5.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest5.SeleniumAmazonUserTest(SeleniumAmazonTest5.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
      </class> <!-- com.thed.zapi.cloud.sample.SeleniumAmazonTest5 -->
    </test> <!-- test5 -->
    <test started-at="2020-02-12T17:08:06Z" name="test2" finished-at="2020-02-12T17:18:52Z" duration-ms="646064">
      <class name="com.thed.zapi.cloud.sample.SeleniumAmazonTest2">
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest2@6b4a4e18]" started-at="2020-02-12T17:08:06Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:10:39Z" duration-ms="153279" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@klmtrips.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-klmtrips]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.seleniumUI(SeleniumAmazonTest2.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.SeleniumAmazonUserTest(SeleniumAmazonTest2.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest2@6b4a4e18]" started-at="2020-02-12T17:10:39Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:11:16Z" duration-ms="36778" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@klmtrips.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-klmtrips]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.seleniumUI(SeleniumAmazonTest2.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.SeleniumAmazonUserTest(SeleniumAmazonTest2.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest2@6b4a4e18]" started-at="2020-02-12T17:11:16Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:11:48Z" duration-ms="31694" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@klmtrips.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-klmtrips]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.seleniumUI(SeleniumAmazonTest2.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.SeleniumAmazonUserTest(SeleniumAmazonTest2.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest2@6b4a4e18]" started-at="2020-02-12T17:11:48Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:12:26Z" duration-ms="37764" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@klmtrips.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-klmtrips]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.seleniumUI(SeleniumAmazonTest2.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.SeleniumAmazonUserTest(SeleniumAmazonTest2.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest2@6b4a4e18]" started-at="2020-02-12T17:12:26Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:13:20Z" duration-ms="54414" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@klmtrips.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-klmtrips]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.seleniumUI(SeleniumAmazonTest2.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.SeleniumAmazonUserTest(SeleniumAmazonTest2.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest2@6b4a4e18]" started-at="2020-02-12T17:13:20Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:14:38Z" duration-ms="77957" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@klmtrips.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-klmtrips]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.seleniumUI(SeleniumAmazonTest2.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.SeleniumAmazonUserTest(SeleniumAmazonTest2.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest2@6b4a4e18]" started-at="2020-02-12T17:14:38Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:15:08Z" duration-ms="29951" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@klmtrips.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-klmtrips]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.seleniumUI(SeleniumAmazonTest2.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.SeleniumAmazonUserTest(SeleniumAmazonTest2.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest2@6b4a4e18]" started-at="2020-02-12T17:15:08Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:15:38Z" duration-ms="29819" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@klmtrips.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-klmtrips]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.seleniumUI(SeleniumAmazonTest2.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.SeleniumAmazonUserTest(SeleniumAmazonTest2.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest2@6b4a4e18]" started-at="2020-02-12T17:15:38Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:16:11Z" duration-ms="33082" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@klmtrips.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-klmtrips]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.seleniumUI(SeleniumAmazonTest2.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.SeleniumAmazonUserTest(SeleniumAmazonTest2.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest2@6b4a4e18]" started-at="2020-02-12T17:16:11Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:16:41Z" duration-ms="30263" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@klmtrips.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-klmtrips]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.seleniumUI(SeleniumAmazonTest2.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.SeleniumAmazonUserTest(SeleniumAmazonTest2.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest2@6b4a4e18]" started-at="2020-02-12T17:16:41Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:17:29Z" duration-ms="47878" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@klmtrips.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-klmtrips]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.seleniumUI(SeleniumAmazonTest2.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.SeleniumAmazonUserTest(SeleniumAmazonTest2.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest2@6b4a4e18]" started-at="2020-02-12T17:17:29Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:18:52Z" duration-ms="83124" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[khalid@klmtrips.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-klmtrips]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.seleniumUI(SeleniumAmazonTest2.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest2.SeleniumAmazonUserTest(SeleniumAmazonTest2.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
      </class> <!-- com.thed.zapi.cloud.sample.SeleniumAmazonTest2 -->
    </test> <!-- test2 -->
    <test started-at="2020-02-12T17:08:06Z" name="test3" finished-at="2020-02-12T17:19:52Z" duration-ms="705929">
      <class name="com.thed.zapi.cloud.sample.SeleniumAmazonTest3">
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@6a01e23]" started-at="2020-02-12T17:08:06Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:11:47Z" duration-ms="220922" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[prashanth@pvrsoft.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sai80080]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-pvrsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.seleniumUI(SeleniumAmazonTest3.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.SeleniumAmazonUserTest(SeleniumAmazonTest3.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@6a01e23]" started-at="2020-02-12T17:11:47Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:12:25Z" duration-ms="37700" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[prashanth@pvrsoft.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sai80080]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-pvrsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.seleniumUI(SeleniumAmazonTest3.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.SeleniumAmazonUserTest(SeleniumAmazonTest3.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@6a01e23]" started-at="2020-02-12T17:12:25Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:13:00Z" duration-ms="35369" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[prashanth@pvrsoft.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sai80080]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-pvrsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.seleniumUI(SeleniumAmazonTest3.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.SeleniumAmazonUserTest(SeleniumAmazonTest3.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@6a01e23]" started-at="2020-02-12T17:13:00Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:13:30Z" duration-ms="29955" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[prashanth@pvrsoft.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sai80080]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-pvrsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.seleniumUI(SeleniumAmazonTest3.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.SeleniumAmazonUserTest(SeleniumAmazonTest3.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@6a01e23]" started-at="2020-02-12T17:13:30Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:13:58Z" duration-ms="27522" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[prashanth@pvrsoft.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sai80080]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-pvrsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.seleniumUI(SeleniumAmazonTest3.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.SeleniumAmazonUserTest(SeleniumAmazonTest3.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@6a01e23]" started-at="2020-02-12T17:13:58Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:15:53Z" duration-ms="115823" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[prashanth@pvrsoft.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sai80080]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-pvrsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.seleniumUI(SeleniumAmazonTest3.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.SeleniumAmazonUserTest(SeleniumAmazonTest3.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@6a01e23]" started-at="2020-02-12T17:15:53Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:16:25Z" duration-ms="31317" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[prashanth@pvrsoft.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sai80080]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-pvrsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.seleniumUI(SeleniumAmazonTest3.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.SeleniumAmazonUserTest(SeleniumAmazonTest3.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@6a01e23]" started-at="2020-02-12T17:16:25Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:18:15Z" duration-ms="110225" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[prashanth@pvrsoft.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sai80080]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-pvrsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.seleniumUI(SeleniumAmazonTest3.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.SeleniumAmazonUserTest(SeleniumAmazonTest3.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@6a01e23]" started-at="2020-02-12T17:18:15Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:19:25Z" duration-ms="70168" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[prashanth@pvrsoft.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sai80080]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-pvrsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.seleniumUI(SeleniumAmazonTest3.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.SeleniumAmazonUserTest(SeleniumAmazonTest3.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest3@6a01e23]" started-at="2020-02-12T17:19:25Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:19:52Z" duration-ms="26868" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[prashanth@pvrsoft.icu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sai80080]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-pvrsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.seleniumUI(SeleniumAmazonTest3.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest3.SeleniumAmazonUserTest(SeleniumAmazonTest3.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
      </class> <!-- com.thed.zapi.cloud.sample.SeleniumAmazonTest3 -->
    </test> <!-- test3 -->
    <test started-at="2020-02-12T17:08:06Z" name="test1" finished-at="2020-02-12T17:21:37Z" duration-ms="811004">
      <class name="com.thed.zapi.cloud.sample.SeleniumAmazonTest1">
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest1@3a079870]" started-at="2020-02-12T17:08:06Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:10:13Z" duration-ms="126413" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@fgtsoft.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-fgtsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.seleniumUI(SeleniumAmazonTest1.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.SeleniumAmazonUserTest(SeleniumAmazonTest1.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest1@3a079870]" started-at="2020-02-12T17:10:13Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:10:41Z" duration-ms="28508" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@fgtsoft.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-fgtsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.seleniumUI(SeleniumAmazonTest1.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.SeleniumAmazonUserTest(SeleniumAmazonTest1.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest1@3a079870]" started-at="2020-02-12T17:10:41Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:11:25Z" duration-ms="43877" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@fgtsoft.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-fgtsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.seleniumUI(SeleniumAmazonTest1.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.SeleniumAmazonUserTest(SeleniumAmazonTest1.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest1@3a079870]" started-at="2020-02-12T17:11:25Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:14:04Z" duration-ms="159314" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@fgtsoft.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-fgtsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.seleniumUI(SeleniumAmazonTest1.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.SeleniumAmazonUserTest(SeleniumAmazonTest1.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest1@3a079870]" started-at="2020-02-12T17:14:04Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:15:21Z" duration-ms="76317" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@fgtsoft.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-fgtsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.seleniumUI(SeleniumAmazonTest1.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.SeleniumAmazonUserTest(SeleniumAmazonTest1.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest1@3a079870]" started-at="2020-02-12T17:15:21Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:15:53Z" duration-ms="32857" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@fgtsoft.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-fgtsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.seleniumUI(SeleniumAmazonTest1.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.SeleniumAmazonUserTest(SeleniumAmazonTest1.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest1@3a079870]" started-at="2020-02-12T17:15:53Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:17:54Z" duration-ms="120746" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@fgtsoft.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-fgtsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.seleniumUI(SeleniumAmazonTest1.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.SeleniumAmazonUserTest(SeleniumAmazonTest1.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest1@3a079870]" started-at="2020-02-12T17:17:54Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:19:43Z" duration-ms="108707" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@fgtsoft.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-fgtsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.seleniumUI(SeleniumAmazonTest1.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.SeleniumAmazonUserTest(SeleniumAmazonTest1.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest1@3a079870]" started-at="2020-02-12T17:19:43Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:20:12Z" duration-ms="29192" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@fgtsoft.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-fgtsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.seleniumUI(SeleniumAmazonTest1.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.SeleniumAmazonUserTest(SeleniumAmazonTest1.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest1@3a079870]" started-at="2020-02-12T17:20:12Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:20:40Z" duration-ms="28201" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@fgtsoft.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-fgtsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.seleniumUI(SeleniumAmazonTest1.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.SeleniumAmazonUserTest(SeleniumAmazonTest1.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest1@3a079870]" started-at="2020-02-12T17:20:40Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:21:09Z" duration-ms="28544" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@fgtsoft.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-fgtsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.seleniumUI(SeleniumAmazonTest1.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.SeleniumAmazonUserTest(SeleniumAmazonTest1.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
        <test-method signature="SeleniumAmazonUserTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.thed.zapi.cloud.sample.SeleniumAmazonTest1@3a079870]" started-at="2020-02-12T17:21:09Z" name="SeleniumAmazonUserTest" finished-at="2020-02-12T17:21:37Z" duration-ms="28263" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@fgtsoft.site]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ram@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[test-sample-fgtsoft]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.seleniumUI(SeleniumAmazonTest1.java:208)
at com.thed.zapi.cloud.sample.SeleniumAmazonTest1.SeleniumAmazonUserTest(SeleniumAmazonTest1.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumAmazonUserTest -->
      </class> <!-- com.thed.zapi.cloud.sample.SeleniumAmazonTest1 -->
    </test> <!-- test1 -->
  </suite> <!-- Parallel Suite -->
</testng-results>
